lib/configOptions.ts

```diff
@@ -51,18 +56,21 @@ const setAllowUsageTracking = async ({ accountId, allowUsageTracking }) => {
 
   updateAllowUsageTracking(isEnabled);
 
-  return logger.log(
-    i18n(`${i18nKey}.allowUsageTracking.success`, { isEnabled })
+  logger.success(
+    i18n(`${i18nKey}.allowUsageTracking.success`, {
+      isEnabled: isEnabled.toString(),
+    })
   );
-};
+}
 
 const ALL_CMS_PUBLISH_MODES = Object.values(CMS_PUBLISH_MODE);
 
-const selectCmsPublishMode = async () => {
-  const { cmsPublishMode } = await promptUser([
+async function selectCmsPublishMode(): Promise<CmsPublishMode> {
+  const { cmsPublishMode } = await promptUser<{
+    cmsPublishMode: CmsPublishMode;
+  }>([
     {
       type: 'list',
-      look: false,
```

Comment:
`look` does not exist as a supported field ([search in repo](https://github.com/search?q=repo%3ASBoudrias%2FInquirer.js%20look&type=code))